core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -29,17 +29,17 @@ object AclCommand {
 
   val Newline = scala.util.Properties.lineSeparator
   val ResourceTypeToValidOperations = Map[ResourceType, Set[Operation]] (
-    Topic -> Set(Read, Write, Describe, All),
-    Group -> Set(Read, All),
-    Cluster -> Set(Create, ClusterAction, All)
+    ResourceType.TOPIC -> Set(Operation.READ, Operation.WRITE, Operation.DESCRIBE, Operation.ALL),
+    ResourceType.GROUP -> Set(Operation.READ, Operation.ALL),
+    ResourceType.CLUSTER -> Set(Operation.CREATE, Operation.CLUSTER_ACTION, Operation.ALL)
   )
 
   def main(args: Array[String]) {
 
     val opts = new AclCommandOptions(args)
 
     if (opts.options.has(opts.helpOpt))
-      CommandLineUtils.printUsageAndDie(opts.parser, "Usage:")
+      CommandLineUtils.printUsageAndDie(opts.parser, description(opts))
```

Comment:
We still may want the "Usage:" string output. Perhaps the description should come after.